Source/Core/Core/IOS/Network/Socket.cpp

```diff
@@ -872,6 +872,9 @@ s32 WiiSockMan::AddSocket(s32 fd, bool is_rw)
     if (setsockopt(fd, SOL_SOCKET, SO_NOSIGPIPE, &opt_no_sigpipe, sizeof(opt_no_sigpipe)) < 0)
       ERROR_LOG_FMT(IOS_NET, "Failed to set SO_NOSIGPIPE on socket");
 #endif
+
+    int opt_broadcast = 1;
+    setsockopt(fd, SOL_SOCKET, SO_BROADCAST, (char*)&opt_broadcast, sizeof(opt_broadcast));
```

Comment:
Notes to self:
 - `SO_BROADCAST` should not be valid _(or should do nothing)_ for TCP sockets
   * SFML set this flags by default for UDP sockets
 - Checking the default options for both new sockets and accepted sockets should be done
 - `errno`/`wsaerror` should be saved/restore to limit side-effects if `SO_TYPE` is used to determine the socket type